src/Uno.UI-vs4mac-iOS-only.sln

```diff
@@ -41,8 +41,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Xaml", "SourceGenerator
 EndProject
```

Comment:
These .sln changes are supposed to be here right?

Reply:
Yes these are intentional, the sln was broken before